website/content/en/docs/ansible/reference/watches.md

```diff
@@ -93,8 +93,7 @@ Some features can be overridden per resource via an annotation on that CR. The o
 | Watching Cluster-Scoped Resources | `watchClusterScopedResources` | Allows the ansible operator to watch cluster-scoped resources that are created by ansible | | false | |
 | Max Runner Artifacts | `maxRunnerArtifacts` | Manages the number of [artifact directories](https://ansible-runner.readthedocs.io/en/latest/intro.html#runner-artifacts-directory-hierarchy) that ansible runner will keep in the operator container for each individual resource. | ansible.operator-sdk/max-runner-artifacts | 20 | |
 | Finalizer | `finalizer`  | Sets a finalizer on the CR and maps a deletion event to a playbook or role | | | [finalizers](../finalizers)|
-
-
+| Selector | `selector`  | Identifies a set of objects based on their labels | | None Applied | [Labels and Selectors] (https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)|
 #### Example
 ```YaML
 ---
```

Comment:
Could you add an example of selector in the doc? See that above has watches examples.